packages/typography/src/Link/Link.styles.ts

```diff
@@ -94,3 +94,8 @@ export const openInNewTabStyles = css`
   left: -1px;
   height: 12px;
 `;
+
+export const inheritSizeStyles = css`
+  font-size: inherit;
+  line-height: inherit;
```

Comment:
I still feel like this should be the default behavior. A link should always be the size of whatever it's inside.
If someone wants to use a Link outside of something that has a font-size set, they can always either provide a `baseFontSize` prop, or add custom CSS